module/VuFindTheme/src/VuFindTheme/ThemeInfo.php

```diff
@@ -222,9 +222,9 @@ public function getMergedConfig(string $key, bool $flatten = false): array
 
         $merged = [];
         while (!empty($currentTheme)) {
-            $currentThemeSet = $arrayFunc(
-                $allThemeInfo[$currentTheme]['mixins'] ?? [],
+            $currentThemeSet = array_merge(
```

Comment:
Did you intentionally revert $arrayFunc to array_merge? If so, I think we can remove the definition of $arrayFunc above since it is no longer used. If not, we need to fix it.

Reply:
Yes this was an intentional change because it was excluding mixin configs when set to array_replace (the other of the two options).